pkg/migrator/core_test.go

```diff
@@ -41,104 +39,129 @@ func newPod(name, namespace string) v1.Pod {
 			Namespace: namespace,
 		},
 	}
-
 }
 
-func newPods(num int) []v1.Pod {
-	var list []v1.Pod
+func newPodList(num int) v1.PodList {
+	var pods []v1.Pod
 	for i := 0; i < num; i++ {
 		pod := newPod(fmt.Sprintf("pod%d", i), fmt.Sprintf("namespace%d", i))
```

Comment:
is it worthwhile to test the nil or empty namespace case?

Reply:
Added.